WalletWasabi.Tests/UnitTests/Transactions/PayjoinTests.cs

```diff
@@ -93,7 +93,7 @@ public void LazyPayjoinServerTest()
 
 			var amount = Money.Coins(0.001m);
 			using Key key = new();
-			PaymentIntent payment = new(key.ScriptPubKey, amount);
+			PaymentIntent payment = new(key.PubKey, amount);
```

Comment:
If I get it correctly, it can be also
```suggestion
			PaymentIntent payment = new(key, amount);
```

to be consistent with `TransactionFactoryTests.cs` changes.